inference-engine/tests/functional/inference_engine/ngraph_reshape_tests.cpp

```diff
@@ -916,8 +916,82 @@ TEST_F(NGraphReshapeTests, ReshapeEDROIFeatureExtractor) {
     Blob::Ptr weights;
     auto network = ie.ReadNetwork(model, weights);
     InferenceEngine::ICNNNetwork::InputShapes newShapes;
-    newShapes["in0"] = {1, 256, 300, 400};
-    newShapes["in1"] = {1000, 256, 7, 7};
+    newShapes["in0"] = {1256, 4};
```

Comment:
Because these are incorrect for this operation. For example, `in0` should have `rank == 2`, not 4: according [the operation specification](https://github.com/openvinotoolkit/openvino/blob/master/docs/MO_DG/prepare_model/convert_model/Legacy_IR_Layers_Catalog_Spec.md#ExperimentalDetectronROIFeatureExtractor), input 0 is 

> 2D input blob describing the rois as 4-tuples: [x1, y1, x2, y2]. Batch size is the number of rois. Coordinates x and y are float numbers and refer to the input image_size. Required.

Also, according to this specification, other inputs are 
> Pyramid of 4D input blobs with feature maps. Batch size must be 1. The number of channels must be the same for all layers of the pyramid.